README.md

```diff
@@ -16,6 +16,8 @@ BitOps is an automated [orchestrator](docs/about.md) for deployment tools using
 
 It leverages a way to describe infrastructure for many environments and IaC tools called an [Operations Repository](docs/operations-repo-structure.md).
 
+<p align="center"><iframe width="100%" height="100%" src="https://www.youtube.com/embed/BiytYu3EefY" title="Intro to BitOps" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
```

Comment:
Iframe doesn't work in the markdown Readme.
![image](https://user-images.githubusercontent.com/1533818/212998757-5d2c60c6-3e32-4a4a-9ef6-8bb6af64240a.png)


You need to create a screenshot and link screenshot to the Youtube URL

Reply:
See https://github.com/stackStorm/st2#stackstorm-overview as en example

Reply:
Works on my local for the preview tho :P 

Yea, np will link as your put above